bin/lib/zopen-update-cacert

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+#
+# Updates the cacert in zopen
+#
+export utildir="$( cd "$(dirname "$0")" >/dev/null 2>&1 && pwd -P )"
+
+. "${utildir}/common.inc"
+
+ZOPEN_CA="${utildir}/../../cacert.pem"
+
+printHelp() 
+{
+  args=$*
+  echo "zopen-update-cacert will update your $ZOPEN_CA file to the latest curl CA certificates extracted from Mozilla"
+}
+
+getCaCertFromCurl()
+{
+  if ! curl -L -o "/tmp/$LOGNAME.cacert.pem" "https://curl.se/ca/cacert.pem"; then
```

Comment:
should have a comment that this is a CA certificate extracted from Mozilla by curl